test-config/wdio.browserstack.conf.js

```diff
@@ -9,27 +9,25 @@ const commonCapabilities = {
   'browserstack.appium_version': '1.14.0'
 }
 
+// https://www.browserstack.com/automate/capabilities
 const allCapabilities = [
-  { ...commonCapabilities, browserName: 'Chrome', browser_version: '89.0', os: 'Windows', os_version: '10' },
-  { ...commonCapabilities, browserName: 'Chrome', browser_version: '74.0', os: 'Windows', os_version: '10' },
+  { ...commonCapabilities, browserName: 'Chrome', browser_version: 'latest', os: 'Windows', os_version: '10' },
```

Comment:
there's no way to set the `latest` os version, so only the browser version is latest.

Reply:
For `Chrome` and `Firefox` we could also test `latest - 1` to make sure changes done to support next lates still work with the previous version.

Reply:
not sure that there's any value in this. Let's say chrome 88 and 89, doesn't seem to be a big difference. The breaking changes usually come in specific releases.

Reply:
I was thinking about the cases where we would need to change the behaviour of live-connect because of changes in the latest browser version. In the current set-up we would test this change against the latest browser version and a very old one - for Chrome and Firefox. This might be enough. On the other side, in my naive assumption, I would say we care about `latest` and `latest - 1` the most, because browser adoption is fast nowadays. But there is no guarantee that we can derive "this works for latest - 1" from looking at either of the versions.

Reply:
I would rather explicitly set the versions that we would like to be tested.